doc/ins_sign_catalyst_registration.md

```diff
@@ -0,0 +1,89 @@
+# Catalyst Voting Key Registration
+
+## Description
+
+Cardano uses a sidechain for its treasury system. One needs to "register" to participate on this sidechain by submitting a registration transaction on the mainnet chain. This CIP details the registration transaction format.
```

Comment:
"This CIP" sounds out of place.

Reply:
yeah, I just copied the first paragraph from the CIP, thanks for noticing